app/bundles/FormBundle/Resources/views/Tokens/tokens_help_bundle.html.twig

```diff
@@ -0,0 +1,36 @@
+<h4 class="mt-lg fw-sb pa-md">{{'mautic.form.forms'|trans}}</h4>
+<p class="pi-md text-helper">{{'mautic.placeholder_tokens.form.redirect_url'|trans}}</p>
+
+<ul id="formTokensAccordion" class="accordion" role="tablist" aria-multiselectable="true">
+    <!-- Form specific tokens -->
+    <li class="panel">
+        <a role="button" id="headingFormTokens" class="accordion-heading collapsed" data-toggle="collapse"
+            data-parent="#formTokensAccordion" href="#collapseFormTokens" aria-expanded="false"
+            aria-controls="collapseFormTokens">
+            <i class="ri-arrow-down-s-line accordion-arrow"></i>
+            <span class="accordion-title">{{ 'mautic.placeholder_tokens.form_tokens'|trans }}</span>
+        </a>
+        <div id="collapseFormTokens" class="collapse accordion-wrapper" role="tabpanel"
+            aria-labelledby="headingFormTokens">
+
+            <table class="table table-hover">
+                <thead>
+                    <tr>
+                        <th>{{ 'mautic.placeholder_tokens.variable_name'|trans }}</th>
+                        <th>{{ 'mautic.placeholder_tokens.variable_syntax'|trans }}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>{{ 'mautic.placeholder_tokens.form.field_value'|trans }}</td>
+                        <td><code>{formfield=alias}</code></td>
+                    </tr>
```

Comment:
As we discussed already, not all tokens can be used everywhere. Form embeds work only on landing pages and maybe in focus items. Not in emails, not in text messages. This will generate a lot of questions about why this form token doesn't load in an email. Or why contactfield tokens don't work in landing pages.

Reply:
That's why each were carefully planned to address this issue, mentioning where people can use what

![image](https://github.com/user-attachments/assets/1bf77a62-466b-4822-b5a8-b5aae09d22eb)

Your input before was really helpful and directed me to specify where things can be used; they're already present in the last version of this PR

![image](https://github.com/user-attachments/assets/f85aa201-d130-4135-b584-31726993dbca)